db/migrate/20160921160155_add_origin_to_fact_values.rb

```diff
@@ -0,0 +1,8 @@
+class AddOriginToFactValues < ActiveRecord::Migration
+  def change
+    add_column :fact_values, :origin, :string, :limit => 255, :default => 'N/A'
```

Comment:
why is it necessary to introduce another column, FactName has STI which determines the fact origin